src/app/modules/group/components/group-manager-list/group-manager-list.component.ts

```diff
@@ -3,11 +3,12 @@ import { GetGroupManagersService, Manager } from '../../http-services/get-group-
 import { GroupData } from '../../services/group-datasource.service';
```

Comment:
I tried removing 3 managers with 1 succeeding request and 2 failing requests. I get the message `"1 manager(s) have been removed, 1 could not be removed."`

Reply:
Probably because requests was sequential. Fixed